demos/controlgroup/index.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
Well, should add them all at once, if we want them.

@scottgonzalez any reason for not having them in the individual demos?